### PR TITLE
Docs/mathnl

### PR DESCRIPTION
**Pull request by jtlap**

### CODE REVIEW

File: include/eve/module/math/regular/expm1.hpp [hunk 1]
```diff
@@ -11,73 +11,60 @@
...
+//! **Parameters**
+//!
+//!   `x`:   [floating real value](@ref eve::floating_real_value).
+//!
+//! **Return value**
+//!
+//!      Returns the [elementwise](@ref glossary_elementwise) exponential input minus one, with good
+//!      accuracy, even for small values of `x`.
+//!
+//!         * If the element is \f$\pm0\f$, \f$\pm0\f$ is returned
```
Comment (jfalcou) [inline include/eve/module/math/regular/expm1.hpp:46]: This code block is badly indented, so the whole text appears as code

File: include/eve/module/math/regular/nthroot.hpp [hunk 2]
```diff
@@ -11,80 +11,61 @@
...
+//! **Parameters**
+//!
+//!   * `x`:   [floating real value](@ref eve::floating_real_value).
+//!
+//!   * `n`:   [integral_value](@ref eve::integral_value). Actually `n` can be a [flint](@ref
+//!   eve::is_flint).
+//!
+//! **Return value**
+//!
+//!    *  Returns [elementwise](@ref glossary_elementwise) the value of \f$x^{1/n}\f$.
```
Comment (jfalcou) [inline include/eve/module/math/regular/nthroot.hpp:46]: Again, we have a single return, so a single sentence is better

File: include/eve/module/math/regular/pow1p.hpp [hunk 3]
```diff
@@ -11,70 +11,55 @@
...
+//!   }
+//!   @endcode
+//!
+//! **Parameters**
+//!
+//!`x`, `y`:   [floating real value](@ref eve::floating_real_value).
+//!
+//! **Return value**
+//!
+//! Returns the [elementwise](@ref glossary_elementwise) power(1+x, y), with good accuracy,
```
Comment (jfalcou) [inline include/eve/module/math/regular/pow1p.hpp:43]: use a formula (1+x)^y

File: include/eve/module/math/regular/gd.hpp [hunk 4]
```diff
@@ -11,74 +11,63 @@
...
+//!
+//! Returns the [elementwise](@ref glossary_elementwise) gudermanian of the input.
+//!
+//! In particular:
+//!
+//!   * If the element is \f$\pm0\f$, 0 is returned.
+//!   * If the element is \f$\pm\infty\f$, \f$\pm\pi/2\f$ is returned.
+//!   * If the element is a `NaN`, `NaN` is returned.
+//!
+//!  auto operator[]( conditional_expression auto cond ) const noexcept;
```
Comment (jfalcou) [inline include/eve/module/math/regular/gd.hpp:51]: We should not have operator[] here

File: include/eve/module/math/regular/asech.hpp [hunk 5]
```diff
@@ -11,75 +11,69 @@
...
+//! The inverse hyperbolic sine is semantically equivalent to   \f$\log(1/x+\sqrt{1/x^2-1})\f$.
+//!
+//! In particular:
+//!
+//!   * If the element is \f$\pm0\f$, \f$\pm\infty\f$ is returned.
+//!   * If the element is \f$1\f$, \f$+0\f$ returned.
+//!   * If the element does not belong to \f$[0,1]\f$, `NaN` is returned.
+//!
+//!
+//!  auto operator[]( conditional_expression auto cond ) const noexcept;
```
Comment (jfalcou) [inline include/eve/module/math/regular/asech.hpp:55]: Remove this and the text below

File: include/eve/module/math/regular/invgd.hpp [hunk 6]
```diff
@@ -11,74 +11,64 @@
...
+//! Returns the [elementwise](@ref glossary_elementwise) inverse
+//! [gudermanian](@ref eve::gd) of the input.
+//!
+//! In particular:
+//!
+//!   * If the element is \f$\pm0\f$, 0 is returned.
+//!   * If the element is \f$\pm\pi/2\f$, \f$\infty\f$ is returned.
+//!   * If the element is not in a  \f$[-\pi/2, \pi/2] \f$, `NaN` is returned.
+//!
+//!  auto operator[]( conditional_expression auto cond ) const noexcept;
```
Comment (jfalcou) [inline include/eve/module/math/regular/invgd.hpp:52]: Remove this text and the one below

File: include/eve/module/math/regular/asec.hpp [hunk 7]
```diff
@@ -11,63 +11,59 @@
...
+//! **Parameters**
+//!
+//!`x`:   [floating real value](@ref eve::floating_real_value).
+//!
+//! **Return value**
+//!
+//! Returns the [elementwise](@ref glossary_elementwise) arc secant of the
+//! input in the range \f$[0 , \pi]\f$.
+//!
+//! IN PARTICULAR:
```
Comment (jfalcou) [inline include/eve/module/math/regular/asec.hpp:46]: DOn't use UPPERCASE LIKE THAT

File: include/eve/module/math/regular/atan2.hpp [hunk 8]
```diff
@@ -11,87 +11,84 @@
...
+//! **Parameters**
+//!
+//!`x`, `y`:   [floating real values](@ref eve::value)
+//!
+//! **Return value**
+//!
+//! the arc tangent of \f$\frac{y}x\f$  in the range \f$[-\pi , +\pi]\f$ radians, is returned.
+//! The **IEEE** limiting values are almost all satisfied :
+//!
+//!     *  If `x` and `y` are both zero or infinites, Nan is returned (this is not standard
```
Comment (jfalcou) [inline include/eve/module/math/regular/atan2.hpp:47]: bad indenting making rendering bad

File: include/eve/module/math/regular/atan2d.hpp [hunk 9]
```diff
@@ -11,87 +11,84 @@
...
+//! **Parameters**
+//!
+//!`x`, `y`:   [floating real values](@ref eve::value)
+//!
+//! **Return value**
+//!
+//! the arc tangent of \f$\frac{y}x\f$  in the range [-180 , +180] radians, is returned.
+//! The **IEEE** limiting values are almost all satisfied :
+//!
+//!     *  If `x` and `y` are both zero or infinites, Nan is returned (this is not standard
```
Comment (jfalcou) [inline include/eve/module/math/regular/atan2d.hpp:47]: Bad indentation -> bad render

File: include/eve/module/math/regular/atan2pi.hpp [hunk 10]
```diff
@@ -11,87 +11,86 @@
...
+//! **Parameters**
+//!
+//!`x`, `y`:   [floating real values](@ref eve::value)
+//!
+//! **Return value**
+//!
+//! the arc tangent of \f$\frac{y}x\f$  in the range [-1, +1] radians, is returned.
+//! The **IEEE** limiting values are almost all satisfied :
+//!
+//!     *  If `x` and `y` are both zero or infinites, Nan is returned (this is not standard
```
Comment (jfalcou) [inline include/eve/module/math/regular/atan2pi.hpp:48]: Bad indentation -> bad rendering

File: include/eve/module/math/regular/expm1.hpp (hunk 1 above)
Comment (jtlap) [reply include/eve/module/math/regular/expm1.hpp:46]: done

File: include/eve/module/math/regular/nthroot.hpp (hunk 2 above)
Comment (jtlap) [reply include/eve/module/math/regular/nthroot.hpp:46]: done

File: include/eve/module/math/regular/pow1p.hpp (hunk 3 above)
Comment (jtlap) [reply include/eve/module/math/regular/pow1p.hpp:43]: done

File: include/eve/module/math/regular/gd.hpp (hunk 4 above)
Comment (jtlap) [reply include/eve/module/math/regular/gd.hpp:51]: done

File: include/eve/module/math/regular/asec.hpp (hunk 7 above)
Comment (jtlap) [reply include/eve/module/math/regular/asec.hpp:46]: done

File: include/eve/module/math/regular/atan2.hpp (hunk 8 above)
Comment (jtlap) [reply include/eve/module/math/regular/atan2.hpp:47]: done

File: include/eve/module/math/regular/atan2d.hpp (hunk 9 above)
Comment (jtlap) [reply include/eve/module/math/regular/atan2d.hpp:47]: done

File: include/eve/module/math/regular/atan2pi.hpp (hunk 10 above)
Comment (jtlap) [reply include/eve/module/math/regular/atan2pi.hpp:48]: done

**Review by jfalcou — APPROVED** [review]

**Pull request merged**